### ISSUE TITLE
cell上添加自定义子视图，下标出错

### ISSUE THREAD

**Issue by iOSchuang** [tigerAndBull/TABAnimated#94]
### New Issue Checklist

* [1] Cell上添加自定义视图之后，该视图上子控件下标不对，无法对其进行设置，如图所示

### Issue Info
<img width="417" alt="git" src="https://user-images.githubusercontent.com/15044460/87220096-b99e7580-c393-11ea-961c-2a116fdd9607.png">

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. iOS / macOS
 Platform Version        | e.g. 12.0 / 10.14.0
 TABAnimated Version      | e.g. 5.0.0 / 4.4.0
 Xcode Version           | e.g. Xcode 11 / Xcode 10
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by tigerAndBull** [comment]
fix on 2.4.4.
